libs/hbb_common/protos/message.proto

```diff
@@ -742,6 +741,7 @@ message Misc {
     TogglePrivacyMode toggle_privacy_mode = 33;
     SupportedEncoding supported_encoding = 34;
   }
+  repeated RdpUserSession rdp_user_sessions = 35;
```

Comment:
```
message RdpUserSessions {
 repeated RdpUserSession rdp_user_sessions = 1;
}
message Misc {
  oneof union {
   ...
    RdpUserSessions rdp_user_sessions = 35;
  }
}
```
You can try this.

Reply:
If I do that I am not able to set rdp user sessions.
<img width="844" alt="Screenshot 2024-02-02 at 1 24 13 PM" src="https://github.com/rustdesk/rustdesk/assets/73148455/8dea0c87-9e40-4671-919a-f321def055e5">

I think it is fine now with having it outside the union as repeated don't work inside it.

Reply:
You can imitate `CaptureDisplays`